spec/ics-002-client-semantics/README.md

```diff
@@ -576,12 +544,13 @@ function commit(
 }
 
 // initialisation function defined by the client type
-function initialize(consensusState: ConsensusState): ClientState {
```

Comment:
I actually prefer to return the `ClientState` and set it to the store on `createClient`

Reply:
at least as I see it, `initialize` is just the constructor of the `ClientState`

Reply:
Hmm. We can do that, I suppose, since this store location does need to be known outside the client-type-specific spec.

Reply:
> Hmm. We can do that, I suppose, since this store location does need to be known outside the client-type-specific spec.

Should we keep the return value then?

Reply:
Yes. Updated.